### PR TITLE
xe-guest install

### PR DESCRIPTION
**Pull request by nagilum99**
Added xe-guest install via webgui

### CODE REVIEW

File: docs/guides.md [hunk 1]
```diff
@@ -21,6 +22,11 @@ echo 'xenguest_enable="YES"' >> /etc/rc.conf.local
 ln -s /usr/local/etc/rc.d/xenguest /usr/local/etc/rc.d/xenguest.sh
 service xenguest start
 ```
+Option 2 is via webgui:  
+Open management page under http(s)://your-configured-ip and go to:
```
Comment (olivierlambert) [inline docs/guides.md:26]: Use Markdown to get a bullet or ordered list, it would be easier to read :+1:

**Review by olivierlambert — CHANGES_REQUESTED** [review]

**Comment by olivierlambert** [comment]
@nagilum99 there's not a lot to fix can you do so we can merge this? If you can, I'll make the modification

File: docs/guides.md (hunk 1 above)
Comment (nagilum99) [reply docs/guides.md:26]: Honestly I don't really know what you want, that it's "easier to read". I like the -> -> -> style, as it shows Step by step what to do. Though it's not a list of points to walk down, it's just the way to get where you need to go.

Comment (olivierlambert) [reply docs/guides.md:26]: No I was thinking:

"You need to do this:
* first thing
* then that
* finally this
"

A bullet list.

**Pull request merged**